integration-tests/pom.xml

```diff
@@ -135,11 +103,6 @@
               <artifactId>java-driver-mapper-processor</artifactId>
               <version>${datastax-java-driver.version}</version>
             </path>
-            <path>
```

Comment:
This won't be required after JAVA-2800, and we can live without it for now (generates a warning).